cogs/handover.py

```diff
@@ -0,0 +1,180 @@
+"""Contains cog classes for handover functionality."""
```

Comment:
I approve the name of the `HandoverCommandCog` but think that the overall `handover.py` file should be renamed to fit better with the fact that it includes all the commands for incrementing the academic year (which includes committee handover, but is not exclusive to that). I'm open to any suggestions to any names that encompass all these commands.

The docstring for this file will require updating accordingly.